webrtc.html

```diff
@@ -655,6 +655,12 @@ <h2>Dictionary <dfn>RTCOfferAnswerOptions</dfn>
                 This option allows the application to provide information about
                 whether it wishes this type of processing enabled or
                 disabled.</p>
+                <div class="issue atrisk">
+                <p>Support for the <code>voiceActivityDetection</code> attribute of
```

Comment:
Chrome implements this as ["When disabled, removes all CN codecs from SDP"](https://cs.chromium.org/chromium/src/third_party/webrtc/pc/media_session.h?type=cs&sq=package:chromium&g=0&l=105) but this should not be needed AFAIK given setCodecPreferences().

I am definitely in support of removing the need for offer options, such that you can rely entirely on parameterless setLocalDescription(), restartIce() and setCodecPreferences(), and have createOffer() be a considered the "old way".